nkms/crypto/kmac.py

```diff
@@ -0,0 +1,34 @@
+import hashlib
+
+
+class KMAC_256(object):
+    # TODO If performance is needed, this could be optimized a bit...
+    # TODO Would be preferable to follow NIST and use cSHAKE.
+    def __init__(self):
+        self.LENGTH_BYTES = 32
```

Comment:
Why are these instance attrs rather than class attrs?

Reply:
Just moving quickly, but I had the idea of ripping this code out and making another lib for it since I couldn't find anything online for a KMAC implementation in Python.

The idea is that you would set the digest `LENGTH_BYTES` arbitrarily based on how you need to use it and `BLOCK_SIZE_BYTES` would be set based on the length you choose for the most secure option.

Reply:
Though, in this case, you are right. Those should be class attributes. 👍